mapeditor/mainwindow.cpp

```diff
@@ -88,6 +89,39 @@ void MainWindow::saveUserSettings()
 	s.setValue(mainWindowPositionSetting, pos());
 }
 
+void MainWindow::parseCommandLine()
```

Comment:
In the very end the functionality should be in a separate tool/ project by itself, that consumes Animation or parts of it from a common place with client and editor.
Can we agree that for first iteration this can stay behind command line options?

Reply:
in the end it's up to you. The risk here is "first iteration" can stay for a very long time